my-app/package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.3",
     "@testing-library/user-event": "^12.6.0",
+    "express": "^4.17.1",
```

Comment:
This should be in its own file, currently the frontend and backend are combined but apparently thats not scale able.
I can go fix this.